### DISCUSSION TITLE
vmware tools are not being installed properly on security onion : offline

### DISCUSSION THREAD

**Discussion by safdar-aneela** [Security-Onion-Solutions/securityonion#1845] [Unsupported Versions] [answered]
Hi,

I am using security onion in import mode as I am only interested in inspecting some pcaps.  I installed vmware tools in order to share a folder with host. Now when I run command

/usr/bin/vm-user 

it says:  could not open /proc/fs/vmblock/dev
I noticed there is no vmblock directory in /proc/fs/

Any suggestions as I am not finding anything useful on internet and kind of stuck.

**Comment by weslambert** [comment, marked as answer]
I'm not sure this is related to Security Onion, but more to VMWare and VMWare tools installation.
